python/cuxfilter/charts/core/non_aggregate/core_line.py

```diff
@@ -134,33 +167,37 @@ def add_range_slider_filter(self, dashboard_cls):
 
         Ouput:
         """
-        if self.stride is None:
-            self.stride = self.stride_type(
-                (self.max_value - self.min_value) / self.data_points
+        if self.x_dtype in CUDF_DATETIME_TYPES:
+            self.filter_widget = pn.widgets.DateRangeSlider(
+                start=self.min_value,
+                end=self.max_value,
+                value=(self.min_value, self.max_value),
+                **{"width": self.width},
+                sizing_mode="scale_width",
+            )
+        else:
+            self.filter_widget = pn.widgets.RangeSlider(
+                start=self.min_value,
+                end=self.max_value,
+                value=(self.min_value, self.max_value),
+                step=self.stride,
+                **{"width": self.width},
```

Comment:
I don't see how this is any different from just passing `width=`
```
In [1]: def foo(a, b, c, d): 
   ...:     print(a, b, c, d) 
   ...:                                                                                                                                                                                                                                         

In [2]: foo(a=1, b=2, c=3, d=4)                                                                                                                                                                                                                 
1 2 3 4

In [3]: foo(a=1, b=2, **{"c": 3}, d=4)                                                                                                                                                                                                          
1 2 3 4
```

Reply:
fixed